### PR TITLE
Creates past events section in events page

### PR DESCRIPTION
**Pull request by twistershark**
closes #193 

## What changed? Why?

This PR creates the past events section in the events page and also creates the events photo slideshow

## How has it been tested?

- [x] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

File: src/components/app/pageEvents/components/eventPhotosSlideshow.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { Children, ReactNode, useCallback, useEffect, useRef, useState } from 'react'
+import { animate, domAnimation, LazyMotion, m, useMotionValue } from 'framer-motion'
+
+import { cn } from '@/utils/web/cn'
+
+const autoplayDelay = 5000
```
Comment (LucasdRossi) [inline src/components/app/pageEvents/components/eventPhotosSlideshow.tsx:6]: We should sue screaming snake case for constants, something like `AUTO_PLAY_DELAY`

File: src/components/app/pageEvents/components/eventDialogContent.tsx [hunk 2]
```diff
@@ -135,9 +136,26 @@ function EventInformation({
...
           </p>
 
-          <GoogleMapsEmbedIFrame address={event.formattedAddress} />
-
-          <EventDialogSocialLinks eventSlug={event.slug} eventState={event.state} />
+          {isPastEvent ? (
+            <EventPhotosSlideshow>
+              {event?.carousel?.map(item => (
+                <NextImage
+                  alt={event.name}
```
Comment (LucasdRossi) [inline src/components/app/pageEvents/components/eventDialogContent.tsx:143]: Can't we add a new field on builder io to better describe the image? Don't think that the event name will describe it well

Comment (twistershark) [reply src/components/app/pageEvents/components/eventDialogContent.tsx:143]: Asked Axel and he said that event name is fine

File: src/components/app/pageEvents/components/eventPhotosSlideshow.tsx (hunk 1 above)
Comment (twistershark) [reply src/components/app/pageEvents/components/eventPhotosSlideshow.tsx]: Updated

**Review by LucasdRossi — APPROVED** [review]

**Review by eduardopicolo-cb — DISMISSED** [review]

**Review by LucasdRossi — APPROVED** [review]

**Review by eduardopicolo-cb — DISMISSED** [review]

File: src/components/app/pageEvents/components/eventPhotosSlideshow.tsx [hunk 3]
```diff
@@ -95,19 +108,21 @@ export function EventPhotosSlideshow({ children }: { children: ReactNode | React
...
-              index !== i && 'bg-white/50',
-            )}
-            key={i}
-            onClick={() => setIndex(i)}
-          />
-        ))}
-      </div>
+      {childrens.length < 15 && (
+        <div className="absolute bottom-4 left-2/4 z-50 flex -translate-x-2/4 gap-2">
+          {new Array(childrens.length).fill('').map((_, i) => (
```
Comment (LucasdRossi) [inline src/components/app/pageEvents/components/eventPhotosSlideshow.tsx:113]:
```suggestion
          {Array.from({ length: childrens.length }).map((_, i) => (
```

**Review by LucasdRossi — DISMISSED** [review]

**Review by LucasdRossi — APPROVED** [review]

**Pull request merged**